### ISSUE TITLE
WCAG: Status messages on our error summary

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#903]
### Description of the bug

If the user is blocked from continuing when clicking "next" on a page with form errors. Screen readers do not explain this to the users. In other words the visual error summary is not announced to users of screen readers. 

Seems like this might be the solution for this: https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19

This criterion only applies to when there no context changes. So not when a new page is loaded for example. 

### Steps To Reproduce

1. Go to an app that has validation rules that stops the users from continuing to the next page
2. Use a screen reader and hear what is announced when trying to click the "next" button
